cmd/dreamboat/main.go

```diff
@@ -200,6 +201,24 @@ var flags = []cli.Flag{
 		Value:   false,
 		EnvVars: []string{"RELAY_FAST_BOOT"},
 	},
+	&cli.StringFlag{
+		Name:    "relay-allow-listed-builder",
+		Usage:   "comma separated list of allowed builder pubkeys",
+		Value:   "",
+		EnvVars: []string{"RELAY_ALLOW_LISTED_BUILDER"},
+	},
+	&cli.DurationFlag{
+		Name:    "relay-submission-limit-rate",
```

Comment:
I think X/s for regular rate and Y burst makes more sense. The default value here should also be 2blocks/second as that is industry standard as seen from the Flashbots relay.